docs/design/ocs-to-ocs-api-server.md

```diff
@@ -0,0 +1,162 @@
+# OCS to OCSs : API Server
+
+## Introduction
+This document defines the API server used for communication between OCS provider and OCS consumer clusters.
+
+
+## Tool
+Following tools can be used to create the API server:
+
+- REST (JSON over HTTP)
+- gRPC
+- OpenAPI3
+
+## Tool Comparison
+### gRPC vs REST
+
+- #### Performance:
+    - gRPC is more performant than REST due to smaller payloads (and other reasons).
```

Comment:
Name the other reasons or remove the statement?

Reply:
I would change it to: gRPC is more performant than REST due to the binary payload

Reply:
I would add that the binary format (protobuf) allows for compile time validation.

Reply:
Updated the verbiage and add some more reasons.